WordPress/gutenberg
# Mobile: Implementing `Clear All Settings` button on Image Settings

## Description

This is a small PR that implements the functionality of the ~`Revert to Original`~ `Clear All Settings` button.
As discussed with Thomas, this button should revert all the editable attributes to their default as when the block is created.

- ~Don't worry about the Image Size option, this will be removed here: https://github.com/WordPress/gutenberg/pull/13750~ (Merged) 

![clear-settings](https://user-images.githubusercontent.com/9772967/52583325-9b93b500-2e2f-11e9-94bf-fc8cc7a9b44b.gif)


To test:
- Check out the [related `gutenberg-mobile` branch](https://github.com/wordpress-mobile/gutenberg-mobile/pull/571) and run the app.
- Add an image block and add an image to it.
- Open the image settings.
- Edit Alt and Link-to.
- Close the BottomSheet.
- Open the image settings again.
- Press `Revert to Original`.
- Check that Alt and Link-to are cleared.
  - The BottomSheet should **not** dismiss automatically.
